tests/cmake_integration/C++11_typecode/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+project(OpenDDS_C++11_Typecode CXX)
+cmake_minimum_required(VERSION 3.8.2)
+
+if(NOT OpenDDS_FOUND)
+  find_package(OpenDDS REQUIRED)
+endif()
+
+if(OPENDDS_NO_OPENDDS_SAFETY_PROFILE EQUAL OFF)
+  message(FATAL_ERROR "This test cannot be built with Safety Profile enabled")
+endif()
+
+set(CMAKE_CXX_COMPILER ${OPENDDS_COMPILER})
+OPENDDS_NO_SUPPRESS_ANYS()
```

Comment:
What's the scope of the change here?  It seems like this belongs in `OPENDDS_TARGET_SOURCES`

Reply:
I agree. It's changing IDL flags. It should either be a global cache variable, a argument to `OPENDDS_TARGET_SOURCES`, or both.

Reply:
ok